### PR TITLE
[Feature/Logging]Rename component, plugins, plugin group

### PR DESCRIPTION
**Pull request by joomdonation**
Pull Request for Issue # .

### Summary of Changes
This PR renames component, plugins and plugin group to a better names as discussed at https://github.com/joomla-projects/privacy-framework/pull/20#issuecomment-390497426

- com_userlogs -> com_actionlogs
- PlgSystemUserLogs -> PlgSystemActionLogs
- PlgActionslogJoomla -> PlgActionlogJoomla

I tested the result myself. @alikon Could you please review and test it and if everything is OK, merge it?

### CODE REVIEW

File: installation/sql/postgresql/joomla.sql [hunk 1]
```diff
@@ -1387,7 +1387,7 @@ INSERT INTO "#__menu" ("id", "menutype", "title", "alias", "note", "path", "link
 (21, 'main', 'com_postinstall', 'Post-installation messages', '', 'Post-installation messages', 'index.php?option=com_postinstall', 'component', 1, 1, 1, 32, 0, '1970-01-01 00:00:00', 0, 1, 'class:postinstall', 0, '', 37, 38, 0, '*', 1),
 (22, 'main', 'com_associations', 'Multilingual Associations', '', 'Multilingual Associations', 'index.php?option=com_associations', 'component', 1, 1, 1, 34, 0, '1970-01-01 00:00:00', 0, 0, 'class:associations', 0, '', 39, 40, 0, '*', 1),
 (23, 'main', 'com_privacy', 'Privacy', '', 'Privacy', 'index.php?option=com_privacy', 'component', 1, 1, 1, 35, 0, '1970-01-01 00:00:00', 0, 0, 'class:privacy', 0, '', 41, 42, 0, '*', 1),
-(24, 'main', 'com_userlogs', 'User Logs', '', 'User Logs', 'index.php?option=com_userlogs', 'component', 1, 1, 1, 36, 0, '1970-01-01 00:00:00', 0, 0, 'class:privacy', 0, '', 43, 44, 0, '*', 1),								    
+(24, 'main', 'com_actionlogs', 'Action Logs', '', 'Action Logs', 'index.php?option=com_actionlogs', 'component', 1, 1, 1, 36, 0, '1970-01-01 00:00:00', 0, 0, 'class:privacy', 0, '', 43, 44, 0, '*', 1),
```
Comment (alikon) [inline installation/sql/postgresql/joomla.sql]: you need to add 483 and 484 the plugin id's  like for mysql

File: administrator/components/com_admin/sql/updates/sqlazure/3.9.0-2018-05-05.sql [hunk 2]
```diff
@@ -1,42 +1,42 @@
 SET IDENTITY_INSERT #__extensions  ON;
 
 INSERT INTO [#__extensions] ([extension_id], [package_id], [name], [type], [element], [folder], [client_id], [enabled], [access], [protected], [manifest_cache], [params], [custom_data], [system_data], [checked_out], [checked_out_time], [ordering], [state]) VALUES
-(36, 0, 'com_userlogs', 'component', 'com_userlogs', '', 1, 1, 1, 1, '', '', '', '', 0, '1900-01-01 00:00:00', 0, 0),
+(36, 0, 'com_actionlogs', 'component', 'com_actionlogs', '', 1, 1, 1, 1, '', '', '', '', 0, '1900-01-01 00:00:00', 0, 0),
 (483, 0, 'plg_system_userlogs', 'plugin', 'userlogs', 'system', 0, 0, 1, 0, '', '{"logDeletePeriod":"0"}', '', '', 0, '1900-01-01 00:00:00', 0, 0),
```
Comment (alikon) [inline administrator/components/com_admin/sql/updates/sqlazure/3.9.0-2018-05-05.sql]: 'plg_system_userlogs'  -> ' plg_system_actionlogs'

File: administrator/components/com_admin/sql/updates/mysql/3.9.0-2018-05-05.sql [hunk 3]
```diff
@@ -1,14 +1,14 @@
 INSERT INTO `#__extensions` (`extension_id`, `package_id`, `name`, `type`, `element`, `folder`, `client_id`, `enabled`, `access`, `protected`, `manifest_cache`, `params`, `custom_data`, `system_data`, `checked_out`, `checked_out_time`, `ordering`, `state`) VALUES
-(36, 0, 'com_userlogs', 'component', 'com_userlogs', '', 1, 1, 1, 1, '', '', '', '', 0, '0000-00-00 00:00:00', 0, 0),
+(36, 0, 'com_actionlogs', 'component', 'com_actionlogs', '', 1, 1, 1, 1, '', '', '', '', 0, '0000-00-00 00:00:00', 0, 0),
 (483, 0, 'plg_system_userlogs', 'plugin', 'userlogs', 'system', 0, 0, 1, 0, '', '{"logDeletePeriod":"0"}', '', '', 0, '0000-00-00 00:00:00', 0, 0),
```
Comment (alikon) [inline administrator/components/com_admin/sql/updates/mysql/3.9.0-2018-05-05.sql]: 'plg_system_userlogs'  -> ' plg_system_actionlogs'

File: administrator/components/com_admin/sql/updates/postgresql/3.9.0-2018-05-05.sql [hunk 4]
```diff
@@ -1,12 +1,12 @@
 INSERT INTO "#__extensions" ("extension_id", "package_id", "name", "type", "element", "folder", "client_id", "enabled", "access", "protected", "manifest_cache", "params", "custom_data", "system_data", "checked_out", "checked_out_time", "ordering", "state") VALUES
-(36, 0, 'com_userlogs', 'component', 'com_userlogs', '', 1, 1, 1, 1, '', '', '', '', 0, '1970-01-01 00:00:00', 0, 0),
+(36, 0, 'com_actionlogs', 'component', 'com_actionlogs', '', 1, 1, 1, 1, '', '', '', '', 0, '1970-01-01 00:00:00', 0, 0),
 (483, 0, 'plg_system_userlogs', 'plugin', 'userlogs', 'system', 0, 0, 1, 0, '', '{"logDeletePeriod":"0"}', '', '', 0, '1970-01-01 00:00:00', 0, 0),
```
Comment (alikon) [inline administrator/components/com_admin/sql/updates/postgresql/3.9.0-2018-05-05.sql]: 'plg_system_userlogs'  -> ' plg_system_actionlogs'

File: installation/sql/sqlazure/joomla.sql [hunk 5]
```diff
@@ -751,7 +751,7 @@ INSERT INTO "#__extensions" ("extension_id", "package_id", "name", "type", "elem
 (33, 0, 'com_fields', 'component', 'com_fields', '', 1, 1, 1, 0, '', '', '', '', 0, '1900-01-01 00:00:00', 0, 0),
 (34, 0, 'com_associations', 'component', 'com_associations', '', 1, 1, 1, 0, '', '', '', '', 0, '1900-01-01 00:00:00', 0, 0),
 (35, 0, 'com_privacy', 'component', 'com_privacy', '', 1, 1, 1, 1, '', '', '', '', 0, '1900-01-01 00:00:00', 0, 0),
-(36, 0, 'com_userlogs', 'component', 'com_userlogs', '', 1, 1, 1, 1, '', '', '', '', 0, '1900-01-01 00:00:00', 0, 0),
+(36, 0, 'com_actionlogs', 'component', 'com_actionlogs', '', 1, 1, 1, 1, '', '', '', '', 0, '1900-01-01 00:00:00', 0, 0),
 (102, 0, 'phputf8', 'library', 'phputf8', '', 0, 1, 1, 1, '', '', '', '', 0, '1900-01-01 00:00:00', 0, 0),
 (103, 0, 'Joomla! Platform', 'library', 'joomla', '', 0, 1, 1, 1, '', '', '', '', 0, '1900-01-01 00:00:00', 0, 0),
 (104, 0, 'IDNA Convert', 'library', 'idna_convert', '', 0, 1, 1, 1, '', '', '', '', 0, '1900-01-01 00:00:00', 0, 0),
```
Comment (alikon) [inline installation/sql/sqlazure/joomla.sql]: you need to add 483 and 484 the plugin id's like for mysql

**Comment by joomdonation** [comment]
@alikon Could you please check my last commit to see whether it fixed all the SQL issues you mentioned?

**Comment by alikon** [comment]
i'm merging it despite the drone cs it can be fixed later in the feature/logging branch

**Pull request merged**